netbout-web/src/main/java/com/netbout/dynamo/DyBout.java

```diff
@@ -121,6 +121,26 @@ public void rename(final String text) throws IOException {
     }
 
     @Override
+    public boolean subscription() throws IOException {
+        return Boolean.parseBoolean(
+            this.item.get(DyFriends.ATTR_SUBSCRIPTION).getS()
```

Comment:
@erimerturk what happens here for existing records, which hasn't the subscription attribute before ?
returns null or what? if so let's add default value `true`